README.md

```diff
@@ -85,8 +85,6 @@ sudo /usr/local/share/gcm-core/uninstall.sh
 
 ---
 
-<!-- this explicit anchor should stay stable so that external docs can link here -->
-<a name="linux-install-instructions"></a>
```

Comment:
Is this actually linked from external docs? If so, is there a commitment not to break this specific link for some reason

Docs (anywhere) can link to headings in markdown using the `url#heading` syntax, as in `docs/wsl.md` in this commit.